src/components/dashboard/Dashboard.js

```diff
@@ -123,6 +126,26 @@ const Dashboard = props => {
     }
   }
 
+  useEffect(() => {
+    if (initDash) {
+      addAnalytics()
+      setInitDash(false)
+    }
+  }, [])
+
+  const addAnalytics = async () => {
+    const { show } = store.get('addWebApp')
+    const isFirstRunOnHomeScreen = await userStorage.userProperties.get('isFirstRunOnHomeScreen')
```

Comment:
rename isFirstRunOnHomeScreen to isAddedToHomeScreen

Reply:
done